h/browser/chrome/lib/options.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hypothesis Extension</title>
+  <style>
+    .hint {
+      font-size: 90%;
+    }
+  </style>
+</head>
+
+<body>
+  <input type="checkbox" id="badge">
+  <label for="badge">Enable extension badge?</label>
+  <p class="hint">
+      By default, the extension button has a small badge that displays the
```

Comment:
This is a good description but I think it would be preferable to shorten it to a single sentence if possible and then link to https://hypothes.is/blog/hypothes-is-chrome-extension-permissions/ for more details.